run_service.sh

```diff
@@ -175,6 +175,92 @@ get_on_chain_service_state() {
     echo "$state"
 }
 
+store=".trader_runner"
+rpc_path="$store/rpc.txt"
+operator_keys_file="$store/operator_keys.json"
+keys_json="keys.json"
+keys_json_path="$store/$keys_json"
+agent_address_path="$store/agent_address.txt"
+service_id_path="$store/service_id.txt"
+service_safe_address_path="$store/service_safe_address.txt"
+store_readme_path="$store/README.txt"
+
```

Comment:
You should also define 2 files for the agent and operator key in standalone format (it's redundant, but it's used by several commands)